### ISSUE TITLE
Theme: Query results show translated patterns side by side.

### ISSUE THREAD

**Issue by StevenDufresne** [WordPress/pattern-directory#273]
**Describe the bug**
I'm not sure if it's enclosed in another ticket but I'll add this in case. 

Since we now have translated patterns, the search results are showing translated versions of the same pattern making the pattern list a bit useless.

**To Reproduce**
1. Go to `wordpress.org/patterns`
2. View the result list, expect to see the same patterns in different languages.

**Expected behavior**
We should only see patterns matching our locale.

**Screenshots**
![](https://d.pr/i/Ws7jH6.png)

**Desktop (please complete the following information):**
 - OS: macOs
 - Browser Chrome
 - Version July 7, latest :)

**Comment by ryelle** [comment]
I had thought there was an issue for this, but it looks like I misread #244, and that's just about the `api.wordpress.org` endpoint. @iandunn, you've been working on this concept for #244, is this on your radar at all?

**Comment by iandunn** [comment]
Ah, no, I was just focused on api.w.org, but should have thought of this too. I'm working on #28 right now, but can circle back to this after that.

**Comment by kjellr** [comment]
This is also happening in the Gutenberg plugin — all translations are showing up at once in the inserter. Should I open a separate issue for this, or will it be resolved by whatever change would fix display in the directory too? 

![patterns](https://user-images.githubusercontent.com/1202812/124974085-61c43a00-dffa-11eb-8459-07beec584952.gif)

**Comment by iandunn** [comment]
The Gutenberg issue should be fixed once https://github.com/WordPress/gutenberg/pull/33052 is merged; it's currently waiting on review.

If you disable Gutenberg and just use 5.8-RC2, you should see the locale-specific results.

**Comment by kjellr** [comment]
Awesome, thanks. 🙌

**Comment by iandunn** [comment]
> circle back to this after that

Nevermind, I'm at a good stopping place w/ the search, so switching to this.
